‏‏assignment2/main.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+from numpy import asarray as ImageAsNumpyArray
```

Comment:
Its confusing, don't rename functions for other libraries. Its sometimes OK to do with your own code. Just use np.asarray

Reply:
Ok